.github/workflows/test-spras.yml

```
@@ -155,6 +156,15 @@ jobs:
         tags: v2
         cache_froms: reedcompbio/py4cytoscape:v2
         push: false
+    - name: Build Local Neighborhood Docker image
+      uses: docker/build-push-action@v1
+      with:
+        path: docker-wrappers/LocalNeighborhood/.
+        dockerfile: docker-wrappers/LocalNeighborhood/Dockerfile
+        repository: sumedhars/local-neighborhood
+        tags: v2
```

```suggestion
        tags: latest
```

If you haven't explicitly tagged your image with a version, the default tag is latest